mirage/views/node.ts

```diff
@@ -1,16 +1,26 @@
 import { HandlerContext, Schema } from 'ember-cli-mirage';
 
+import Contributor from 'ember-osf-web/models/contributor';
+import Node from 'ember-osf-web/models/node';
+import User from 'ember-osf-web/models/user';
+
+import { Root } from '../factories/root';
+
 export function createNode(this: HandlerContext, schema: Schema) {
     const attrs = this.normalizedRequestAttrs();
     const now = (new Date()).toISOString();
-    const node = schema.nodes.create({
+    const node = schema.nodes.create<Node>({
```

Comment:
How long do you think until you figure this one out?

Reply:
Is that a challenge?! :smile:
*@jamescdavis disappears into a hole*